Sources/Sentry/include/SentrySDK.h

```diff
@@ -140,6 +140,11 @@ SENTRY_NO_INIT
  */
 @property (nonatomic, class, readonly) BOOL crashedLastRun;
 
+/**
+ * Checks if the SentrySDK was enabled or not.
+ */
+@property (nonatomic, class, readonly) BOOL isEnabled;
```

Comment:
It's important that there is a clean API for suspending and resuming the Sentry client, in order to smoothly conform with the privacy wishes that users have for the app & comply with GDPR and related privacy legislation.  Consequently, I believe there should be a setter for this property.